guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p>which guide ?, no reference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Reference cannot be linked until Pix2Pix has been merged.</p>